PyDSS/utils/utils.py

```diff
@@ -20,6 +20,12 @@
 logger = logging.getLogger(__name__)
 
 
+def check_redirect(file_name):
+    result = dss.run_command("Redirect {}".format(file_name))
+    if result != '':
+        raise Exception(f'Redirect failed {result}')
```

Comment:
Can you include the filename in the exception string?  OpenDSS may not report anything about it if failure happens in a secondary redirect.